app/src/main/res/values/strings.xml

```diff
@@ -107,8 +107,15 @@
     <string name="str_cnt_mnu_share">Share</string>
     <string name="str_cnt_mnu_delete">Delete</string>
     <string name="title_activity_trustees">Trustees</string>
+    <string name="activity_trustees_text">Add the Comrade Numbers here</string>
+    <string name="hint_comrade1">Comrade 1</string>
```

Comment:
You don't have to define a new string for this comrade 1 is already there on the list.<string name="comrade1">Comrade 1</string>